lib/Optimizer/Transforms/LiftArrayAlloc.cpp

```diff
@@ -182,8 +217,8 @@ class AllocaPattern : public OpRewritePattern<cudaq::cc::AllocaOp> {
     if (std::distance(alloc->getUses().begin(), alloc->getUses().end()) < size)
       return false;
 
-    // Keep a scoreboard for every element in the array. Every element *must* be
-    // stored to with a constant exactly one time.
+    // Keep a scoreboard for every element in the array. Every element *must*
+    // be stored to with a constant exactly one time.
```

Comment:
clang-format appears to have unformatted this along the way. (Should revert this one, it appears.)

Reply:
this is done by the clang format though :)

Reply:
run clang format again